.github/workflows/build_documentation.yml

```
@@ -16,7 +16,7 @@ jobs:
       install_libgl1: true
       package: diffusers
       notebook_folder: diffusers_doc
-      languages: en ko zh
+      languages: en ko zh jp
```

The ISO 639-1 language code for Japanese is `ja`

```suggestion
      languages: en ko zh ja
```